examples/go/kontain-http-uname/skaffold.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: skaffold/v1beta1
```

Comment:
for my information what is this file used for?

Reply:
Its actually a google project for devs to easily use Kubernetes.

"Skaffold handles the workflow for building, pushing and deploying your application, allowing you to focus on what matters most: writing code." Auto-build, deploy to k8s. I haven't fully tested and enabled it yet on all projects, but its used by devs to run apps on k8s easily on change.